### PR TITLE
DOC: A few fixes in examples.

### PR DESCRIPTION
**Pull request by arokem**
Primarily, not using 'jet' in these two examples.

Also: a call to `plotting.show()` was raising an error in plot_probabilistic*
so this fixes that.

### CODE REVIEW

**Comment by bthirion** [comment]
LGTM. The example looks much better now.

File: examples/connectivity/plot_probabilistic_atlas_extraction.py [hunk 1]
```diff
@@ -50,7 +50,12 @@
 # Display the correlation matrix
 from matplotlib import pyplot as plt
 plt.figure(figsize=(10, 10))
-plt.imshow(correlation_matrix, interpolation="nearest")
+# Mask out the major diagonal
+correlation_matrix[range(correlation_matrix.shape[-1]),
+                   range(correlation_matrix.shape[-1])] = 0
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_probabilistic_atlas_extraction.py]: This is simpler and faster: `np.fill_diagonal(correlation_matrix, 0)`

File: examples/connectivity/plot_probabilistic_atlas_extraction.py [hunk 2]
```diff
@@ -64,4 +69,4 @@
 plotting.plot_connectome(correlation_matrix, coords,
                          edge_threshold="80%")
 
-plotting.show()
+plt.show()
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_probabilistic_atlas_extraction.py]: We use `plotting.show()` on purpose. In a lot of examples, this allows the user to display the plot without importing explicitely pyplot. We'd like to keep it this way.

File: examples/connectivity/plot_signal_extraction.py [hunk 3]
```diff
@@ -51,7 +51,12 @@
 # Plot the correlation matrix
 from matplotlib import pyplot as plt
 plt.figure(figsize=(10, 10))
-plt.imshow(correlation_matrix, interpolation="nearest")
+# Mask the main diagonal for visualization:
+correlation_matrix[range(correlation_matrix.shape[-1]),
+                   range(correlation_matrix.shape[-1])] = 0
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_signal_extraction.py]: Same remark here.

File: examples/connectivity/plot_signal_extraction.py [hunk 4]
```diff
@@ -67,8 +72,13 @@
 
 correlation_matrix = np.corrcoef(time_series.T)
 
+# Mask the main diagonal for visualization:
+correlation_matrix[range(correlation_matrix.shape[-1]),
+                   range(correlation_matrix.shape[-1])] = 0
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_signal_extraction.py]: And here also.

**Comment by AlexandreAbraham** [comment]
I requested some cosmetic changes. Otherwise, +1, this really improves the example indeed.

**Comment by arokem** [comment]
Thanks for the comments. Hopefully I got everything?

On Sun, Nov 8, 2015 at 12:17 AM, Alexandre Abraham <notifications@github.com

**Comment by AlexandreAbraham** [comment]
That was fast. LGTM too, merging.

**Comment by AlexandreAbraham** [comment]
Thanks!

**Pull request merged**